apps/web/src/components/chat/chat-input.tsx

```diff
@@ -47,48 +55,62 @@ export function ChatInput() {
     [],
   );
 
+  const showStackPicker = isHomePage(pathname);
+
   return (
-    <div className="w-full border border-dashed border-input bg-background text-black flex items-center relative px-4 lg:px-6 gap-2 md:gap-4">
-      <Textarea
-        className="flex-1 py-[18px] bg-transparent border-none outline-none text-foreground placeholder:text-sm md:placeholder:text-base resize-none"
-        placeholder={
-          isAppPage(pathname)
-            ? 'Type your message...'
-            : 'Describe the app you want to build...'
-        }
-        value={inputValue}
-        onChange={handleChange}
-        onKeyDown={(e) => {
-          if (e.key === 'Enter' && !e.shiftKey) {
-            e.preventDefault();
-            handleSubmit();
+    <div className="flex-col py-[18px]  w-full border border-dashed border-input bg-background text-black flex items-center relative px-4 lg:px-6 gap-2 md:gap-4">
```

Comment:
```suggestion
    <div className="flex-col py-[18px]  w-full border border-dashed border-input bg-background text-black flex items-center relative px-4 lg:px-6 gap-2 md:gap-4">
```

py-[18px]? a bit of a magical one no? could we try to stick with relative values?

Reply:
changed to py-4.5